Realm/RLMSyncPermissionResults.mm

```diff
@@ -64,8 +63,6 @@ bool keypath_is_valid(NSString *keypath)
 
 /// Sort by the Realm Object Server path to the Realm to which the permission applies.
 RLMSyncPermissionSortProperty const RLMSyncPermissionSortPropertyPath       = @"path";
-/// Sort by the identity of the user to whom the permission applies.
-RLMSyncPermissionSortProperty const RLMSyncPermissionSortPropertyUserID     = @"userId";
```

Comment:
Was this also removed from the header? I don't see a removal when searching for `RLMSyncPermissionSortPropertyUserID` in this diff.

Reply:
You are right; I missed it. Turns out I was only searching for occurrences of that string inside the tests group.